v2.3.4 pre-production checklist (#118)
wp-media/rocket-lazy-load: readme.txt

Requires at least: 4.7

> Tested up to: 5.3

Reviewer: ```suggestion
Tested up to: 5.5
```

The element must have the class `rocket-lazyload`, and a `data-bg` attribute, which value is the CSS url for the image.

== Changelog ==
= 2.3.4 =

> Bugfix: Fatal error from League Container conflict with WooCommerce 4.4

Reviewer: ```suggestion
Bugfix: Prevent a Fatal error related to the League Container package conflict with WooCommerce 4.4
```

> Bugfix: Prevent a Fatal error related to the League Container package conflict with WooCommerce 4.4

Reviewer: ```suggestion
Enhancement: Allow `<a>` tags to lazyload background images
Enhancement: Add <noscript> tag to lazyloaded picture elements
Bugfix: Prevent a Fatal error related to the League Container package conflict with WooCommerce 4.4
Bugfix: Update lazyload for background images support for new version of lazyload script
Bugfix: Correctly apply the rocket-lazyload class on elements with a background-image and an empty class value Bugfix: Correctly apply the rocket-lazyloadclass on elements with malformed HTML
Bugfix: Prevent a display issue with background-images when using different types of quotes around the URL 
Bugfix: Prevent Layout from breaking when <img> alt attribute has any html encoded characters
```
